consequence_prediction/repeat_expansion_variants/pipeline.py

```diff
@@ -23,6 +23,14 @@ def none_to_nan(*args):
     return [np.nan if a is None else a for a in args]
 
 
+def get_variant_identifier(measure):
```

Comment:
I'm not sure about using the term identifier in this: there has been a few attempts to capture the "variant identifier" so far we settled on the simple def of `chrom_pos_REF_ALT` (see [here](https://github.com/EBIvariation/eva-opentargets/blob/b5e96f8c626dbe02d269565177d8f047382eea8f/eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py#L232)).
Maybe prefer the boring but accurate `get_variant_name_or_hgvs`